webrender/build.rs

```diff
@@ -35,6 +34,9 @@ const VK_EXTENSIONS: &'static str = "#extension GL_ARB_shading_language_420pack
                                      #extension GL_ARB_explicit_attrib_location : enable\n\
                                      #extension GL_ARB_separate_shader_objects : enable\n";
 
+// https://www.khronos.org/registry/vulkan/specs/1.1-extensions/html/vkspec.html#features-limits
+const MAX_VERTEX_INPUT_ATTRIBUTES: u32 = 16;
```

Comment:
nit: this is a bit redundant. Could be `MAX_VERTEX_ATTRIBUTES` or `MAX_INPUT_ATTRIBUTES`